lexis/data/Progress.h

```diff
@@ -0,0 +1,38 @@
+
+/* Copyright (c) 2016, Human Brain Project
+ *                     Stefan.Eilemann@epfl.ch
+ */
+
+#ifndef LEXIS_DATA_PROGRESS_H
+#define LEXIS_DATA_PROGRESS_H
+
+#include <lexis/api.h>
+#include <lexis/data/detail/progress.h> // base class
```

Comment:
Was done by the original author. lexis/progess.h is this file, what other name would you suggest?